src/pages/Categories/CategoriesBeta.vue

```diff
@@ -45,6 +45,20 @@
 					/>
 				</div>
 			</kv-grid>
+			<kv-grid class="tw-bg-secondary tw-pt-5 tw-mb-5 tw-rounded">
```

Comment:
The button should have 48px of padding top and bottom, there's too much space here, I think you need to remove the `tw-pt-5` from this kv-grid

Reply:
I've been playing around with inspecting the screen for sizing. It seems like tw-pt-6 is good for the padding size of 48px, but I made the button itself smaller so that everything fits Nathan's dimensions. 

Before I push again, there's a nice part of KvButton that allows you to choose the state. When I choose active it's always the secondary color like in Figma, but when I choose default it's white and on hover it's secondary. I like it with the default state because you can actually see the button changing when you hover. When it's always active it seems like it doesn't really work. 

Should I just keep it active to match Figma?